extension/main.js

```diff
@@ -23,10 +23,10 @@
 			if (interval !== null && interval !== parseInt(window.GitHubNotify.settings.get('interval'), 10)) {
 				window.GitHubNotify.settings.set('interval', interval);
```

Comment:
Hmm good point.
I noticed that `window.GitHubNotify.settings.get('interval')` can return a string that can't be parsed as an integer. Most notably when a fresh install it can return `"undefined"`. It would be good to be defensive about this somehow.

I'll put in a commit that address that more closely.